account/identity_blackbox_test.go

```diff
@@ -86,6 +90,28 @@ func (s *IdentityRepositoryTestSuite) TestOKToDelete() {
 	}
 }
 
+func randString(n int) string {
+	const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letterBytes[rand.Int63()%int64(len(letterBytes))]
+	}
+	return string(b)
+}
+
+func (s *IdentityRepositoryTestSuite) TestOKToObfuscate() {
+	// given
+	identity := createAndLoadUserAndIdentities(s)
```

Comment:
We have text fixtures for this.
Can you please remove `createAndLoadUserAndIdentities` function?
You can use
```go
fxt := tf.NewTestFixture(s.T(), s.DB, tf.Identities(1), tf.Users(1))
```
You can use `fxt.Identities[0]` and `fxt.Users[0]` in your test.